freqtrade/rpc/webhook.py

```diff
@@ -77,13 +82,28 @@ def send_msg(self, msg: Dict[str, Any]) -> None:
     def _send_msg(self, payload: dict) -> None:
         """do the actual call to the webhook"""
 
-        try:
-            if self._format == 'form':
-                post(self._url, data=payload)
-            elif self._format == 'json':
-                post(self._url, json=payload)
-            else:
-                raise NotImplementedError('Unknown format: {}'.format(self._format))
-
-        except RequestException as exc:
-            logger.warning("Could not call webhook url. Exception: %s", exc)
+        success = False
+        attempts = 0
+        while not success and attempts <= self._retries:
+            if attempts:
+                if self._retry_delay: time.sleep(self._retry_delay)
+                logger.info("Retrying webhook...")
+
+            attempts += 1
+
+            try:
+                if self._format == 'form':
+                    response = post(self._url, data=payload)
+                elif self._format == 'json':
+                    response = post(self._url, json=payload)
+                elif self._format == 'raw':
+                    response = post(self._url, data=payload['data'], headers={'Content-Type': 'text/plain'})
+                else:
+                    raise NotImplementedError('Unknown format: {}'.format(self._format))
+            
+                """throw a RequestException if the post was not successful"""
```

Comment:
Please use comments  (`# ...`) instead of multiline strings to comment.

While this seems to be the same, it's actually an executable line (which will be executed every time) - which is really unnecessary for comments.

Reply:
I normally would, but I saw everything else in that file was using """ so I was just sticking with the convention.